### DISCUSSION TITLE
Version number on hover

### DISCUSSION THREAD

**Discussion by Mystqr4** [WindowTop/WindowTop-App#263] [Ideas]
It would be nice if the system tray icon could show the version number on hover or even a direct link to the about screen on right click or a check for update option (even though it's not always in line with the updates here. haha) on right click within the menu.

**Comment by gileli121** [comment]
thanks for the suggestion.
I am aware of the check updates issue in the desktop version.
Sometimes I do it when I am not 100% sure that the version is stable, so I don't send the notification so fast.

Anyway, this is not how it works in the Microsoft Store version.
The store version is always aware of the latest version and will trigger the update notification when a new version is released.

I can solve this in 2 ways:
1) Fix the problem in the regular version (not the Microsoft Store one), so it will always show an update notification after the release
Or - if we want the same behavior also in the store version:
2) Fix the problem in the Microsoft Store version so it will not show update notifications when I am not 100% about the stability.

My feeling is to go with option 1.
Most of the time, users report problems, and I ask the user to test the fix for a few days.
And most of the time, there are no risky updates because the app is relatively stable.
A lot of times, I just don't push the notification because I forgot about it.

The solution is to use GitHub API, so GitHub itself will handle the update notifications without the need to do it.
Maybe I need to make some workflow here.

**Comment by Mystqr4** [comment]
Another idea is if it would possible for users to sign up/check an option to get updates from the github releases. I understand they are sort of a pre-release version as such so maybe that would work also so it becomes an option for people and they are aware they are downloading pre-release versions and stability may not be 100% on these versions.

Either way even if just the ability to see the version number on the icon in the system tray would be a huge plus.

**Reply by gileli121** [reply]
Thanks for the suggestion.
It seems like the best idea to use Github API for this.
